src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -17,6 +17,7 @@
 )
 
 from sqlfluff.core.dialects import load_raw_dialect
+from sqlfluff.core.parser.grammar.greedy import StartsWith
```

Comment:
For ANSI this is added to the `sqlfluff.core.parser` import above. Any reason this needs it's own one here?

Though actually I don't think you need this, based on two comments I made below:

Reply:
Ahh sorry, VScode does auto-import for me when I use and unimported class/function and tends to choose the source file to import from (rather than the __init__.py in src/sqlfluff/core/parser).

As you said, it's unnecessary though so I'll remove it 👍